modules/virt-organize-vms-web.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * virt/managing_vms/virt-list-vms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="virt-listing-vms-cli_{context}"]
```

Comment:
Did you mean to include `cli` in the id, rather than `web`? The procedure seems to be for the web console.